database/migration/changesets/20200315_CreateWorkloadTable.xml

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd
+                        http://www.liquibase.org/xml/ns/dbchangelog
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+    <changeSet author="tbl@broadinstitute.org"
+               dbms="postgresql"
+               id="20200315"
+               logicalFilePath="20200315_CreateWorkloadTable.xml">
+        <comment>
+            Define a new root workload table schema.
+            There is a row in this table for every workload.
+            The 'table' column here names another table.
+            That other table has the composite Postgres type 'workflow',
+            and is named with the 'pipeline' column
+            composed with the 'id' column of this table.
+        </comment>
+        <rollback>
+            I have no idea.
```

Comment:
🤔 what is supposed to be in this block?

Reply:
It's spozed to be how to rollback the changset.
I should just delete those, since I don't think they matter.
Liquibase does not support automated rollback for SQL blocks.